backend/MASZ/Repositories/AutoModerationEventRepository.cs

```diff
@@ -152,5 +167,25 @@ await DiscordAPI.FetchUserInfo(c.UserId, CacheBehavior.OnlyCache)
             }
             return filteredEvents;
         }
+        private async Task CreateAndLinkEvidence(ModCase modCase, VerifiedEvidence evidence)
+        {
+            try
+            {
+                IMessage msg = await DiscordAPI.GetIMessage(evidence.ChannelId, evidence.MessageId, CacheBehavior.IgnoreButCacheOnError);
```

Comment:
what if the message is not fetched? you should add a null check here.

Reply:
This is covered by the try catch, but I see how that may not be a good practice, so I'll add the check